packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -186,13 +200,13 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
         <Modal isOpen={true} isModeless={true} dragOptions={DRAG_OPTIONS} styles={floatingLocalVideoModalStyle}>
           {localParticipant && defaultOnRenderLocalVideoTile}
         </Modal>
-        <GridLayout styles={styles}>{defaultOnRenderRemoteParticipants}</GridLayout>
+        <GridLayout styles={styles as any}>{defaultOnRenderRemoteParticipants}</GridLayout>
```

Comment:
What's the reason as any needed added here? Can this be avoided?

Reply:
So the reason is that styles takes in a particular type which can not be undefined or null.
I can either assign styles a default `{}` or add an assignment here like
`styles={styles ?? {}}`
Based on a prev comment by @prprabhu-ms showing concern around this approach causing re-renders,  I decided to move away from default assignments and trick the typing system.

Let me know if you think of a better approach.

Reply:
We should always avoid any as we no longer will get compiler checks. Probably easiest option is above the method define `const emptyStyles = {}` then do `styles={styles ?? emptyStyles}`

Reply:
Good advice, thanks